RecoBTag/DeepFlavour/src/sorting_modules.cpp

```diff
@@ -0,0 +1,29 @@
+
+
+
+#include "../interface/sorting_modules.h"
```

Comment:
relative paths are not allowed

Reply:
changed with https://github.com/cms-sw/cmssw/pull/19893/commits/246c352e4a0a3714d93ccefe6524f387224f13b9